parm/archive/master_gefs.yaml.j2

```diff
@@ -0,0 +1,14 @@
+# Set variables/lists needed to parse the gefs templates
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+
+datasets:
+{% filter indent(width=4) %}
+{% if DO_EXTRACTVARS %}
```

Comment:
@WalterKolczynski-NOAA After checking through the yamls, I think the `DO_EXTRACTVARS` condition can be removed here. All the products being archived are either from the `atmos_prod`, `ensstat`, `ocean_prod`, `ice_prod ` or the wave post tasks.  None of the products being archived are from the `extractvars` task.